Tools/scripts/deepfreeze.py

```diff
@@ -407,6 +407,9 @@ def main() -> None:
         source = f.read()
     modname = args.module or os.path.basename(args.file).removesuffix(".py")
     output = args.output or modname + ".c"
+    outdir = os.path.dirname(output)
+    if outdir:
+        os.makedirs(outdir, exist_ok=True)
```

Comment:
That's one way to solve the missing directory in OOT builds. :)

The correct, designated way is a bit more obscure. The ``configure.ac`` file has a list of source directories that are populated in OOT builds. You have to add the directory to ``SRCDIRS`` block.

Reply:
Okay, I'll bite. Why is the obscure way better?